src/org/opendatakit/briefcase/automation/AutomationConfiguration.java

```diff
@@ -0,0 +1,58 @@
+package org.opendatakit.briefcase.automation;
+
+import java.nio.file.Path;
+import java.util.Optional;
+
+public class AutomationConfiguration {
```

Comment:
Since we're not using this class I'd suggest to remove it but we know we will eventually need one. What we don't really know is if those will be the members it will hold.

I suggest we leave just the scriptLocation and remove the rest for now.

Reply:
Ok